README.md

```diff
@@ -57,7 +59,7 @@ resources:
   source:
     uri: https://github.com/cloudfoundry-community/simple-go-web-app.git
 
-- name: resource-deploy-web-app
```

Comment:
Yeah that makes sense.I guess it's just confusing as the actual config on the `put` in a plan ends up specifying the app (through the manifest reference) so it always felt like the resource represented a space. Anyways I'll change this.